how-to/install-frontend-packages.rst

```diff
@@ -0,0 +1,118 @@
+.. _install-frontend-packages:
+
+How to add arbitrary frontend packages to a project
+===================================================
+
+The process for installing and configuring frontend packages to a project is simply a particular
+case of the :ref:`process to install system-level packages <install-system-packages>` - i.e.
+using ``RUN`` to execute the relevant commands in the ``Dockerfile``.
+
+This document will use the installation of as an example, and will assume a project that uses our
+base project version 3.23.
+
+
+What we want to do
+------------------
+
+If we were installing these components by hand we might run:
+
+..  code-block:: bash
+
+    # Activate Node Version Manager using nvm.sh. Note that the directory $NVM_DIR is set as
+    # an environment variable by the base project.
+    source $NVM_DIR/nvm.sh
+
+    # Install Node version 6.10.1.
+    nvm install 6.10.1
+
+    # Set a default Node version to be used in any new shell.
+    nvm alias default 6.10.1
+
+    # Use the default Node version now.
+    nvm use default
+
+    # Ensure that the correct version of Node Package Manager is globally installed.
+    npm install -g npm@"6.10.1"
```

Comment:
no need for quotes here

Reply:
Is that because the version number is a literal, unlike the `npm install -g npm@"$NPM_VERSION"` example below?

Reply:
yup